torch/_ops.py

```diff
@@ -925,6 +945,94 @@ def tags(self):
     # TODO: add more methods to expose information about input and output arguments
 
 
+class PythonDispatcherOpOverload(OpOverload):
+    def __repr__(self):
+        return "<PythonDispatcherOpOverload(op='{}.{}', overload='{}')>".format(
+            *self._schema.name.split("::"), self._overloadname
+        )
+
+    def _fallthrough_keys(self) -> list[DispatchKey]:
+        # TODO: we should be calling the fallback for these, but a fallthrough is almost close
+        # enough to the fallback in most cases that we care about.
+        _DEFAULT_FALLTHROUGH_KEYS = [
+            DispatchKey.ADInplaceOrView,
+            DispatchKey.BackendSelect,
+            DispatchKey.PythonTLSSnapshot,
+            DispatchKey.PythonDispatcher,
+        ]
```

Comment:
we should try to model fallthroughs closer to how the C++ dispatcher does it. That is, the operator doesn't come with fallthrough keys, instead these are fallbacks.

Reply:
(maybe in a follow-up)